scripts/aggregate-changelogs/config.yaml

```diff
@@ -16,51 +16,89 @@ repositories:
     category: Management API
   giantswarm/azure-admission-controller:
     category: Management API
```

Comment:
I have updated the apps from the current `managed `catalog` and removed playground since we don't support anymore